### PR TITLE
Fixes small erros in the code samples

### PR DESCRIPTION
**Pull request by cies**
Now that I have your attention:

* frickin' love MeiliSearch!
* ~~the API likes queries as POSTs (especially when facet params are given), this is against the HTTP spec and has some potential issues (with caching for instance); it would be great to explain this in the docs somewhere and/or find a way to do all retrieve queries as GETs~~
* the operators (i.e.: `=`, `!=`, etc.) are not clearly explained when it comes to facets on array properties, like: what does `lang != "pt"` do on `lang = ["pt", "pt-BR"]`?
* consider dropping the `$` at the start of each line in the bash/curl examples (makes 'm more copy pastable)

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
-    -X GET 'http://localhost:7700/indexes/movies'
+  $ curl -X GET 'http://localhost:7700/indexes/movies'
 list_all_indexes_1: |1-
-  $ curl \
-    -X GET 'http://localhost:7700/indexes'
+  $ curl -X GET 'http://localhost:7700/indexes'
 create_an_index_1: |-
-  $ curl \
-    -X POST 'http://localhost:7700/indexes' \
+  $ curl 'http://localhost:7700/indexes' \
```
Comment (eskombro) [inline .code-samples.meilisearch.yaml:6]:
```suggestion
  $ curl -X POST 'http://localhost:7700/indexes' \
```

File: .code-samples.meilisearch.yaml [hunk 2]
```diff
@@ -27,8 +24,7 @@ get_documents_1: |-
   $ curl \
     -X GET 'http://localhost:7700/indexes/movies/documents?limit=2'
 add_or_replace_documents_1: |-
-  $ curl \
-    -X POST 'http://localhost:7700/indexes/movies/documents' \
+  $ curl 'http://localhost:7700/indexes/movies/documents' \
```
Comment (eskombro) [inline .code-samples.meilisearch.yaml:27]:
```suggestion
  $ curl -X POST 'http://localhost:7700/indexes/movies/documents' \
```

File: .code-samples.meilisearch.yaml [hunk 3]
```diff
@@ -37,8 +33,7 @@ add_or_replace_documents_1: |-
         "release_date": "2019-03-23"
     }]'
 add_or_update_documents_1: |-
-  $ curl \
-    -X POST 'http://localhost:7700/indexes/movies/documents' \
+  $ curl 'http://localhost:7700/indexes/movies/documents' \
```
Comment (eskombro) [inline .code-samples.meilisearch.yaml:36]:
```suggestion
  $ curl -X POST 'http://localhost:7700/indexes/movies/documents' \
```

File: .code-samples.meilisearch.yaml [hunk 4]
```diff
@@ -51,17 +46,15 @@ delete_one_document_1: |-
   curl \
     -X DELETE 'http://localhost:7700/indexes/movies/documents/25684'
 delete_documents_1: |-
-  curl \
-    -X POST 'http://localhost:7700/indexes/movies/documents/delete-batch' \
+  curl 'http://localhost:7700/indexes/movies/documents/delete-batch' \
```
Comment (eskombro) [inline .code-samples.meilisearch.yaml:49]:
```suggestion
  curl -X DELETE 'http://localhost:7700/indexes/movies/documents/delete-batch' \
```

**Review by eskombro — CHANGES_REQUESTED** [review]
Hello! Just passing by to say that I see a lot of `-X POST` and `-X DELETE` options have been removed in this PR, and this will probably brake several samples. I added some suggestions to show where the problem may be, but they are definitely not exhaustive!

**Comment by fharper** [comment]
Thanks for your first PR @cies 🎉

When you have time, please fix the issues highlighted by @eskombro and once done, @react-learner will review the PR.

**Comment by react-learner** [comment]
Hey @cies , I'm closing this PR for now. If you'd like to continue working on it again, feel free to reopen it!

**Pull request closed without merging**